### ISSUE TITLE
how to train?

### ISSUE THREAD

**Issue by isalirezag** [tonylins/pytorch-mobilenet-v2#14]
Can you please tell us how can we train the model from scratch? and also how can we test it?

**Comment by AIROBOTAI** [comment]
Same question. Unless we know exactly the hyperparameters used by the author, we cannot use the code at all.

**Comment by basaltzhang** [comment]
I move the MobilenetV2.py into this project. https://github.com/miraclewkf/MobileNetV2-PyTorch
 lr = 0.045, gamma=0.92, epoch = 99. batchsize = 320
After training one epoch the validation accuracy is 0.7207
